src/main/host/thread.rs

```diff
@@ -259,6 +259,13 @@ impl TryFrom<libc::pid_t> for ThreadId {
     }
 }
 
+impl From<ProcessId> for ThreadId {
+    fn from(value: ProcessId) -> Self {
+        // A process ID is also a valid thread ID
+        ThreadId(value.into())
+    }
+}
```

Comment:
If shadow's process ID and thread ID are to just be equivalent to Linux's pid, I think it would make sense to delete the `ThreadId` and just use `ProcessId`? I think being able to convert between `ThreadId` and `ProcessId` like this can be confusing.

Reply:
I think it's useful to have two distinct types to avoid accidentally confusing them, but occasionally we need to convert between them.